test/data/test_reference_files/exiv2-bug1145c.exv.out

```diff
@@ -98,7 +98,7 @@ Exif.Photo.SceneCaptureType                  Short       1  0  Standard
 Exif.Photo.Contrast                          Short       1  0  Normal
 Exif.Photo.Saturation                        Short       1  0  Normal
 Exif.Photo.Sharpness                         Short       1  0  Normal
-Exif.Photo.LensSpecification                 Rational    4  0/10 0/10 0/10 0/10  0/10 0/10 0/10 0/10
+Exif.Photo.LensSpecification                 Rational    4  0/10 0/10 0/10 0/10  0mm F0

```

Comment:
Again, I would bother printing 0mm...

Maybe any 0 for both focal and aperture should translate to "unknown" or "unspecified"?

Reply:
@kmilos 
I noticed the all zeros problem too but wasn't sure of the correct translated output. If the tag doesn"t apply, we could use `n/a`. However if we consider the zeros value inaccurate, then the output options are the vanilla value in `( )` or `Undefined`. I checked in the Exif specification but this didn't give any guidance.